src/infoPanel/infoPanel.c

```diff
@@ -155,6 +198,10 @@ int main(int argc, char *argv[])
 	SDL_Event event;
 
 	while (!quit && wait_confirm) {
+		uint32_t ticks = SDL_GetTicks();
+		acc_ticks += ticks - last_ticks;
+		last_ticks = ticks;
+
 		while (SDL_PollEvent(&event)) {
 			if (event.type == SDL_KEYDOWN)
 			{
```

Comment:
Can't we put feature: `left/right != quit` in this PR?

Reply:
Added in [7c54eb2](https://github.com/OnionUI/Onion/pull/106/commits/7c54eb262359702bf66c39920290a5133a2e80db)